### DISCUSSION TITLE
injection failed

### DISCUSSION THREAD

**Discussion by spaghetti443** [winstxnhdw/lc-hax#192] [Q&A]
When i open the game and open launch.bat my game crashes and this message comes up
![Screenshot 2024-01-07 160210](https://github.com/winstxnhdw/lc-hax/assets/86524366/e567b30a-6d2c-4292-be56-353c58c0ca96)

**Comment by GigglingMinister** [comment]
I thought it just me

**Comment by winstxnhdw** [comment]
might be my fault. can you send me your current commit hash?

**Reply by GigglingMinister** [reply]
where exactly to write git rev-parse HEAD in

**Reply by winstxnhdw** [reply]
nvm, i think i found the issue. i've pushed a fix. please try again.

**Reply by GigglingMinister** [reply]
works now

**Reply by winstxnhdw** [reply]
thanks for the update

**Reply by spaghetti443** [reply]
ill try now

**Reply by spaghetti443** [reply]
yep works now ty
